scripts/score-project-quality-ai.py

```
@@ -550,30 +1045,166 @@ def main():
+
+    def _with_budget(d, it):
+        b, e = it.get('budgetAmount') or 0, it.get('execAmount') or 0
+        tops = top_recipients.get(str(it['pid'])) or {}
+        oc = outcomes.get(str(it['pid'])) or {}
+        return {**d,
+                'indicators': oc.get('indicators', []),
+                'indicatorCounts': oc.get('indicatorCounts', {}),
+                'review': oc.get('review'),
+                'logicModel': oc.get('logicModel'),
+                'relatedProjects': oc.get('relatedProjects', []),
+                'logicModel': oc.get('logicModel'),
```

Ruff (0.15.21)

[error] Dictionary key literal `'logicModel'` repeated

Remove repeated key literal `'logicModel'`

(F601)